evap/staff/forms.py

```diff
@@ -391,3 +360,18 @@ class Meta:
         model = FaqQuestion
         exclude = ("section",)
 
+
+class TextAnswerForm(forms.ModelForm, BootstrapMixin):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['original_answer'].widget.attrs['readonly'] = "True"
+
+    class Meta:
+        model = TextAnswer
+        fields = ("original_answer", "reviewed_answer",)
+
+    def clean_reviewed_answer(self):
+        reviewed_answer = self.cleaned_data.get('reviewed_answer')
+        if reviewed_answer == self.instance.original_answer or reviewed_answer == '':
```

Comment:
according to https://code.djangoproject.com/ticket/19251 our friends the linebreaks can be different in some situations. to be safe i'll add the normalization again.